cli.js

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env node
 import { globbySync } from 'globby'
 import fs from 'node:fs'
+import getStdin from 'get-stdin'
```

Comment:
Is this dependency fully necessary? It's effectively
```
const buffs = []
for await (const chunk of process.stdin) buffs.push(chunk)
Buffer.concat(buffs)
```

Reply:
It's on me, https://github.com/keithamus/sort-package-json/pull/296#discussion_r1170780711

I thought you would prefer deps.

Reply:
I don't mind particularly either way, but as you've already suggested the dep, let's continue with it.